Orso.Arpa.Api/Controllers/SectionsController.cs

```diff
@@ -26,9 +26,9 @@ public SectionsController(ISectionService sectionService)
         [Authorize(Policy = AuthorizationPolicies.HasRolePolicy)]
         [HttpGet]
         [ProducesResponseType(StatusCodes.Status200OK)]
-        public async Task<ActionResult<IEnumerable<SectionDto>>> Get()
+        public async Task<ActionResult<IEnumerable<SectionDto>>> Get([FromQuery]bool instrumentsOnly = false)
```

Comment:
what's the idea behind _instrumentsOnly_? When looking into the SectionSeedData, e.g. a Trumpet is flagged as an instrument, a Flugelhorn or a Cornet is not.

Reply:
We distinguish between main instrument (selectable from all sections with the IsInstrument Flag set to true) and doubling instruments which can be all of the child sections of the main instrument.